csharp/test/Apache.Arrow.Compression.Tests/Apache.Arrow.Compression.Tests.csproj

```
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="xunit" Version="2.4.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageReference Include="coverlet.collector" Version="1.2.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Apache.Arrow\Apache.Arrow.csproj" />
+    <ProjectReference Include="..\..\src\Apache.Arrow.Compression\Apache.Arrow.Compression.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="Resources\ipc_lz4_compression.arrow" />
+    <EmbeddedResource Include="Resources\ipc_lz4_compression.arrow" />
+    <None Remove="Resources\ipc_lz4_compression.arrow_stream" />
+    <EmbeddedResource Include="Resources\ipc_lz4_compression.arrow_stream" />
+    <None Remove="Resources\ipc_zstd_compression.arrow" />
+    <EmbeddedResource Include="Resources\ipc_zstd_compression.arrow" />
+    <None Remove="Resources\ipc_zstd_compression.arrow_stream" />
+    <EmbeddedResource Include="Resources\ipc_zstd_compression.arrow_stream" />
```

```suggestion
    <EmbeddedResource Include="Resources\ipc_lz4_compression.arrow" />
    <EmbeddedResource Include="Resources\ipc_lz4_compression.arrow_stream" />
    <EmbeddedResource Include="Resources\ipc_zstd_compression.arrow" />
    <EmbeddedResource Include="Resources\ipc_zstd_compression.arrow_stream" />
```

Are the `None Remove`s necessary?

---

No, I don't think there's any need for these, thanks